spec/sidekiq_flow/adapters/set_storage_spec.rb

```diff
@@ -0,0 +1,480 @@
+require 'spec_helper'
+
+RSpec.shared_examples '.start_workflow common' do
+  it 'should store the workflow in redis' do
+    expect {
+      subject
+    }.to change { $redis.keys.count }.from(0).to(2)
+
+    expect( $redis.keys.first ).to eq('in-progress')
```

Comment:
in-progress is our set key, 

should check for `$redis.sismember(IN_PROGRESS, 'workflow-key-here')`

Reply:
There is no workflow key in this block, did `expect( $redis.smembers(IN_PROGRESS).count ).to eq(1)` instead

Reply:
Changed. In the end it's : `expect( redis.sismember(IN_PROGRESS, SidekiqFlow::Client.find_workflow_key(workflow.id)) ).to eq(true)`